srtp-ekt-diet/draft-ietf-perc-srtp-ekt-diet.md

```diff
@@ -403,24 +404,24 @@ applied for each received packet.
    first N bytes from the srtp\_master\_salt field.
 
 5. If the SSRC in the EKTPlaintext does not match the SSRC of the SRTP
-   packet, then all the information from this EKTPlaintext MUST be
-   discarded and the following steps in this list are not done.
+   packet received, then all the information from this EKTPlaintext MUST be
+   discarded and the following steps in this list are aborted.
```

Comment:
I think "skipped" might be a better term than "aborted" but I am OK with aborted if you think it is better